security/scute/Portfile

```
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                scute
+
+version             1.7.0
+revision            0
+categories          security
+maintainers         {@gouttegd dgouttegattat:incenp.org}
```

You should also mark this port a `openmaintainer`, unless there is a reason why only you and you alone should be allowed to make changes and updates, no matter how minor to this port.

```suggestion
maintainers         {@gouttegd dgouttegattat:incenp.org} \
                    openmaintainer
```

---

I realise I also misunderstood the convention for obfuscating the email address (`domain:name` instead of `name:domain`). Sorry about that, I’ll fix that as well.